### PR TITLE
corrected some comment spelling issue

### PR DESCRIPTION
**Pull request by pryhmez**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by pryhmez** [comment]
i have no errors

File: app/src/main/java/swati4star/createpdf/model/BrushItem.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 public class BrushItem {
     private final int mColor;
+    private
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/model/BrushItem.java]: why this?

File: app/src/main/res/layout/activity_preview.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
     <View
         android:id="@+id/view"
         android:layout_width="match_parent"
-        android:layout_height="1dp"
+        android:layout_height="2dp"
```
Comment (Swati4star) [inline app/src/main/res/layout/activity_preview.xml]: why are you changing this?

File: app/src/main/java/swati4star/createpdf/util/BottomSheetUtils.java [hunk 3]
```diff
@@ -28,7 +28,7 @@ public void populateBottomSheetWithPDFs(BottomSheetPopulate listener) {
     /**
      * Retrieves a list of available excel files on the device
      * @param listener a bottom sheet listener used to inform the caller when the list of files
-     * is available
+     * are available
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/BottomSheetUtils.java]: is this only the intended chnage?

**Comment by Swati4star** [comment]
Closing as invalid PR.

**Pull request closed without merging**